src/data/agile-quiz.ts

```diff
@@ -75,34 +81,49 @@ const agileQuiz = [
   },
   {
     Question:
-    "Which of the following is NOT one of the four core principles of the Agile methodology?",
-  Answer: "Comprehensive documentation over working software",
-  Distractor1: "Individuals and interactions over processes and tools",
-  Distractor2: "Responding to change over following a plan",
-  Distractor3: "Customer collaboration over contract negotiation",
-  Explanation:
-    "Instead of detailed specifications and planning, Agile prioritizes shipping working software and making incremental changes based on feedback.",
-  Link: "https://www.freecodecamp.org/news/agile-methods-and-methodology-for-beginners/",
-},
-{
-  Question:
-    "Which of the following is NOT an Agile framework?",
-  Answer: "Waterfall",
-  Distractor1: "Scrum",
-  Distractor2: "Kanban",
-  Distractor3: "XP (Extreme Programming)",
-  Explanation:
-    "Waterfall is an opposing project management methodology that has little to no room for feedback or iteration until the product is completely built and tested.",
-  Link: "https://www.freecodecamp.org/news/agile-methods-and-methodology-for-beginners/",
-},
+      "Which of the following is NOT one of the four core principles of the Agile methodology?",
+    Answer: "Comprehensive documentation over working software",
+    Distractor1: "Individuals and interactions over processes and tools",
+    Distractor2: "Responding to change over following a plan",
+    Distractor3: "Customer collaboration over contract negotiation",
+    Explanation:
+      "Instead of detailed specifications and planning, Agile prioritizes shipping working software and making incremental changes based on feedback.",
+    Link: "https://www.freecodecamp.org/news/agile-methods-and-methodology-for-beginners/",
+  },
+  {
+    Question: "Which of the following is NOT an Agile framework?",
+    Answer: "Waterfall",
+    Distractor1: "Scrum",
+    Distractor2: "Kanban",
+    Distractor3: "XP (Extreme Programming)",
+    Explanation:
+      "Waterfall is an opposing project management methodology that has little to no room for feedback or iteration until the product is completely built and tested.",
+    Link: "https://www.freecodecamp.org/news/agile-methods-and-methodology-for-beginners/",
+  },
+  {
+    Question: "In the Agile framework, what is a user story?",
+    Answer:
+      "A simple description of a software feature from the perspective of a user or customer",
+    Distractor1:
+      "A simple description of a software feature from the perspective of a developer",
+    Distractor2:
+      "A simple description of a software feature from the perspective of the Scrum Master",
+    Distractor3:
+      "A simple description of a software feature from the perspective of the Product Owner",
+    Explanation:
+      "A user story is an informal, general explanation of a software feature written from the perspective of the end user.",
+    Link: "https://www.atlassian.com/agile/project-management/user-stories",
```

Comment:
Whenever possible we always want to push freeCodeCamp resources.

Let's use this link instead

```suggestion
    Link: "https://www.freecodecamp.org/news/how-to-write-user-stories-epics-pesonas/",
```

Reply:
Thanks for suggesting an appropriate freeCodeCamp resource, Jessica. I've updated the link now.